Source/9_Resources/BSML/Leaderboard/Components/Header/CaptorClan.bsml

```diff
@@ -0,0 +1,9 @@
+﻿<!-- TODO: How do you make the clanTag smaller than the pref-height of the container it's in?-->

```

Comment:
`child-expand-height="false"` and `child-control-height="false"`
https://docs.unity3d.com/ru/2019.4/Manual/script-VerticalLayoutGroup.html
https://monkeymanboy.github.io/BSML-Docs/Tags/verticallayouttag/

Reply:
Fixed